requirements_production.txt

```diff
@@ -1,10 +1,11 @@
 # Requirements for OpenSlides in production in alphabetical order
-bleach>=1.5.0,<1.6
-channels>=1.1,<1.2
+bleach>=1.5.0,<2.2
+channels<1.2
+daphne<2
 Django>=1.10.4,<1.11
```

Comment:
Why do you not updated to django 2?

Reply:
I tried. It breaks a lot of stuff. Even a small update to django 1.11 breaks some mutables and immutables which cause some trouble.

Reply:
Ok, see new issue #3582